runtime/src/stakes.rs

```diff
@@ -14,7 +17,6 @@ use {
             self,
             state::{Delegation, StakeActivationStatus, StakeState},
         },
-        stake_history::StakeHistory,
```

Comment:
Pretty nice! The only visible change to stakes.rs is where to get `StakeHistory` from.